airflow/models.py

```diff
@@ -3529,7 +3522,8 @@ def pickle_info(self, session=None):
             d['pickle_len'] = len(pickled)
             d['pickling_duration'] = "{}".format(datetime.now() - dttm)
         except Exception as e:
-            logging.debug(e)
+            log = LoggingMixin().logger
```

Comment:
Why is this _not_ using self.logger? The logging mixin has been removed form this class which seems odd.

Reply:
Maken sense to have a logger available in DAG. Preferably with the name of the DAG appended for the things it is logging

Reply:
The reason I pointed this case out is that it's a method on DAG, and it did have the LoggingMixin, but we removed that in this PR (lines -2756/+2749)

Reply:
Yes, now I think of it. It allows also the users to re-use the dag's logger to emit logs statements.

Reply:
Good observation.

https://issues.apache.org/jira/browse/AIRFLOW-1602
https://github.com/apache/incubator-airflow/pull/2602